site/docs/components/card/usage.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Card component
```

Comment:
There seems to be inconsistency using this heading. Maybe @jpmorganchase/salt-content can advice wether we keep it or not? In either case we'll need to amend the rest of the docs